views/lib/view.js

```diff
@@ -46,6 +49,28 @@ export default class View {
      * @type {string}
      */
     this.creditsText = ''
+
+    this.initialized = false
+  }
+
+  /**
+   * Converts an InflectionData, returned from an inflection tables library, into an HTML representation of an inflection table.
```

Comment:
again, minor but can we update the comment to reflect what is actually happening now? It's not producing HTML anymore but a data object of rows and columns, correct?